TCMSApp/src/client/app/landing/landing.route.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('app.landing')
+        .run(appRun);
+
+    appRun.$inject = ['routerHelper'];
+    /* @ngInject */
+    function appRun(routerHelper) {
+        routerHelper.configureStates(getStates());
+    }
+
+    function getStates() {
+        return [
+            {
+                state: 'index',
+                config: {
+                    url: '/index',
+                    templateUrl: 'app/landing/landing.html',
+                    controller: 'LandingController',
+                    controllerAs: 'vmLanding',
+                    title: 'Landing',
+                    settings: {
+                        nav: 4,
+                        content: '<i class="fa fa-lock"></i> Defects'
```

Comment:
nav: 4,

content: '<i class="fa fa-lock"></i> Defects'


Do you know what is it?